src/rvwmo.tex

```diff
@@ -71,6 +71,8 @@ \subsection*{Memory Model Primitives}
 
 \begin{commentary}
   In the memory model literature, the term ``RCpc'' stands for release consistency with processor-consistent synchronization operations, and the term ``RCsc'' stands for release consistency with sequentially-consistent synchronization operations~\cite{Gharachorloo90memoryconsistency}.
+  
+While there are many different informal defintions of these annotations, in the context of the RISC-V RVWMO (and by extension RVTSO) these terms are concisely and completely defined axiomatixally by Preserved Program Order rules 4, 5 and 6 on page ##.
```

Comment:
Can you add a section link or something instead of “page ##”?

Reply:
"defintions" should probably be definitions